config/config.toml

```diff
@@ -241,6 +244,8 @@ scroll-button = 8
 middle-emulation = true
 left-handed = false
 natural-scroll = false
+disable-with-typing = true
```

Comment:
Please name this `disable-while-typing` instead of `-with-`, to match the libinput name

Reply:
I don't know where `with` came from. Changed.